roles/packages/tasks/main.yml

```diff
@@ -20,9 +21,10 @@
       - unattended-upgrades
       - vim
     state: latest
-    update_cache: yes
-    autoremove: yes
-    autoclean: yes
+    update_cache: true
```

Comment:
Booleans are truthy

Reply:
Hum both works, must of examples in the documentation use `yes` instead of `true`